adoc/retail_chap_introduction.adoc

```diff
@@ -4,29 +4,29 @@
 include::entities.adoc[]
 
 
-
-[abstract]
---
+[[retail.sect.intro.what]]
+== What is {smr}?
 
 {smr} {productnumber} is an open source infrastructure management solution, optimized and tailored specifically for the retail industry.
+It uses the same technology as {susemgr}, but is customized to address the needs of retail organizations.
 
-This document provides an overview of {smr}, and guides you through initial installation and setup.
-It should be read in conjunction with the {susemgr} documentation suite, available from https://www.suse.com/documentation/suse-manager-3/
+It is designed for use in retail situations where customers can use point-of-service terminals to purchase or exchange goods, take part in promotions, or collect loyalty points.  
```

Comment:
Maybe just add "e.g." or "for example" or "among others" to state that this list is not exhaustive.